app/src/models/popup.ts

```diff
@@ -121,3 +124,12 @@ export type Popup =
       pullRequest: PullRequest
     }
   | { type: PopupType.MergeConflicts; repository: Repository }
+  | {
+      type: PopupType.OversizedFiles
+      dispatcher: Dispatcher
```

Comment:
We have the `dispatcher` already available in `app.tsx` so we don't need to provide this explicitly